javascript/javascript1/week2/script.js

```diff
@@ -82,4 +82,37 @@ const clothesWhenCold = getWhatToWear(11);
 
 console.log(clothesWhenHot);
 console.log(clothesWhenOk);
-console.log(clothesWhenCold);
\ No newline at end of file
+console.log(clothesWhenCold);
+
+
+// Student manager
+
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (class07Students.length >= 6 && studentName !== "Queen") {
+        return "Cannot add more students to class 07";
+    } else if (class07Students.includes(studentName)) {
+        return `Student ${studentName} is already in the class`;
+    } else if (studentName === "") {
+        return "You cannot add an empty string to a class";
+    } else {
+        class07Students.push(studentName);
+    }
+}
+
+function getNumberOfStudents() {
```

Comment:
This function doesn't work. 
hint : When you find out the issue, you might fix it in the other function as well. Weird right!! It works in the other one but not here.